packages/e2e-tests/specs/editor/various/preview.test.js

```diff
@@ -54,7 +54,7 @@ async function waitForPreviewNavigation( previewPage ) {
  * @param {boolean} shouldBeChecked If true, turns the option on. If false, off.
  */
 async function toggleCustomFieldsOption( shouldBeChecked ) {
-	const baseXPath = '//*[contains(@class, "edit-post-preferences-modal")]';
+	const baseXPath = '//*[contains(@class, "interface-preferences-modal")]';
```

Comment:
Off-topic, just jogging my thoughts down: A good example of having class names in e2e selectors isn't a good idea. We could probably just do `[role="dialog"]` here, since there should be only one dialog/modal usable at any given time. It's not very readable what this modal is though, so an accessible role selector is still useful: `role=dialog[name="Preferences"]`